docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "3"
+services:
+  db:
+    image: postgres:latest # You can specify a version, e.g., postgres:13
+    restart: always
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
```

Comment:
I think the `.env.example` db string will need to be updated to match?

Reply:
Correct. I will fix it

Reply:
done

Reply:
thank you